app/scripts/lib/router.js

```diff
@@ -228,22 +227,6 @@ function (
       return viewOptions;
     },
 
-    _checkForRefresh: function (currentView) {
-      var refreshMetrics = this.storage.get('last_page_loaded');
-      var screenName = currentView.getScreenName();
-
-      if (refreshMetrics && refreshMetrics.view === screenName && this.metrics) {
```

Comment:
I notice that here we used to check `this.metrics` but in the code extracted to `page-refresh-watch.js` we don't bother. Is it okay to assume that it will always be set?